src/main/java/dev/vavelin/example/liquint/demo/ui/GetDepartmentEndpoint.java

```diff
@@ -0,0 +1,24 @@
+package dev.vavelin.example.liquint.demo.ui;
```

Comment:
General comment.
I would significantly simplify the application. What you need is the endpoint only that queries the database.
For the sake of the simplicity you can even expose the repository as REST (it is possible with JPA).
See this: [https://spring.io/guides/gs/accessing-data-rest/]
The project showcases the intests backed by the liquibase and the main purpose is to present how we can dynamically create and destroy schemas running the intests automating the db setup.

Reply:
Simplified